### PR TITLE
cloud_function: add alarmMissingVAAs

### PR DESCRIPTION
**Pull request by panoel**
Not sure how environment variables work in cloud functions.  If there needs to be code changes, I haven't done them.

Also, calling myself out here:  `CHAIN_INFO_MAP` is in 2 different consts files.  But they are slightly different.  Is there a better way?

### CODE REVIEW

File: cloud_functions/src/alarmMissingVaas.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  } catch (e) {
+    console.log('could not get missing VAAs');
+    res.sendStatus(500);
+  }
+  return;
+}
+
+function formatMessage(msg: ObservedMessage): string {
+  let displayInfo: DisplayInfo = {
+    txHash: msg.txHash,
```
Comment (bruce-riley) [inline cloud_functions/src/alarmMissingVaas.ts:38]: Could you add the chainName here to make it easier for humans to understand?

Comment (panoel) [reply cloud_functions/src/alarmMissingVaas.ts]: That and more...

File: common/src/consts.ts [hunk 2]
```diff
@@ -110,3 +133,285 @@ export const CIRCLE_DOMAIN_TO_CHAIN_ID: { [key: number]: ChainId } = {
...
+    explorerStem: `https://basescan.org`,
+  },
+  32: {
+    name: 'sei',
+    evm: false,
+    chainId: CHAIN_ID_SEI,
+    endpointUrl: '',
+    explorerStem: '',
+  },
+  33: {
```
Comment (evan-gray) [inline common/src/consts.ts:327]: This ID doesn't seem right. did you mean 3104

File: dashboard/src/utils/consts.ts [hunk 3]
```diff
@@ -276,6 +277,14 @@ export const CHAIN_INFO_MAP: { [key: string]: CHAIN_INFO } = {
     explorerStem: '',
     icon: seiIcon,
   },
+  33: {
```
[the remark this answers is not in the document]
Comment (evan-gray) [reply dashboard/src/utils/consts.ts:280]: actually, should this whole thing just be imported from the consts `CHAIN_INFO_MAP`? 🤔

**Review by evan-gray — CHANGES_REQUESTED** [review]
good start, couple small things in the dashboard 🙂

File: dashboard/src/utils/consts.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (panoel) [reply dashboard/src/utils/consts.ts]: Put most of this in common/consts.ts and created new structure for icon images.

**Pull request merged**